03_probability_intro/03_probability_intro.jl

```diff
@@ -13,126 +13,338 @@ macro bind(def, element)
+# ╔═╡ 2feab95a-925c-11eb-0959-ad38cc3ddae6
+md"
+## Joint probability
+
+We refer to joint probability as the probability of two events occurring together and at the same time. 
+Let's see how it works with an example.
+
+Imagine we want to study this two events:
+
+R= “Today It will rain”
+
+L = “The number 39 will win the lottery”
+
+Let's analyze the nature of these events a little bit, are they related? 
+Technically, we can think everything in our planet is interconnected with everything else, but in practice, it is fair to assume these two events are pretty independent from one another.
+In a more formal way, what this is telling us is that knowing the probability of one of the events does not give us information about the probability of the other event. 
+That is the definition of independence in this context. 
+
+In the language of probabilities, we can write a special property for independent events.
+
+$P(R and L) = P(R) * P(L)$
+
+Colloquially, this means that the probability of both events “Today It will rain” and “The number 39 will win the lottery” happening together is equal to the product of each of the probabilities of each one happening individually.
+ 
+When the events are not independent from one another, we can use the conditional probability of events we saw earlier.
+
+Now, lets study the joint probability of this two events:
+
+R= “Today It will rain”
+
+H= “humidity will exceed 50%”
+
+If it is raining, there is a high probability that the humidity levels will rise, so the probability of humidity exceeding 50% will be affected.
+
+To calculated this joint probability we use:
+
+$P(H\text{ and }R) = P(R)P(H|R)$
+
+
+For two general events A and B, this is:
 
 $P(A\text{ and }B) = P(A)P(B|A)$
 
 With this formula in mind, and another property of conjunction probability,
 
 $P(A\text{ and }B) = P(B\text{ and }A)$
 
-which just means what we naturally interpret of two events happening *at the same time*. If the two events do happen at the same time, it doesn't matter if we write $B\text{ and }A)$ or $A\text{ and }B$, setting an order in the expression is just a consequence of having to write it. Just as like doing $2 + 3$ or $3 + 2$, the 'and' logical operator is commutative. We now proceed to derive the famous **Bayes' theorem**! We write 
+which just means what we naturally interpret of two events happening at the same time.
+If the two events do happen at the same time, it doesn't matter if we write $B\text{ and }A)$ or $A\text{ and }B$, setting an order in the expression is just a consequence of having to write it. 
+Just like doing $2 + 3$ or $3 + 2$, the 'and' logical operator is commutative.
+"
+
+# ╔═╡ 39a5328a-9267-11eb-2cb4-4181b250b0b8
+md"## Bayes theorem 
+
+With this concepts in mind we will now proceed to derive the famous Bayes' theorem
+
 
 $P(B\text{ and }A) = P(B)P(A|B),$
 
-and putting all the pieces together, 
```

Comment:
why delete this line? It is good to articulate the mathematical expressions with some words